dao/seeds/source_types.yml

```diff
@@ -143,6 +154,20 @@ azure:
         isRequired: true
         validate:
         - type: required
+    - type: provisioning_lighthouse_subscription_id
+      name: Provisioning — Subscription ID
+      fields:
+        - component: text-field
+          name: authentication.authtype
+          hideField: true
+          initializeOnMount: true
+          initialValue: provisioning_lighthouse_subscription_id
+        - component: text-field
+          name: authentication.username
+          label: Subscription ID
+          isRequired: true
+          validate:
+            - type: required
```

Comment:
Can confirm this works fine on our end, assuming there are no other changes from the existing type other than the type/name field :-)